chart/validator-plugin-oci/values.yaml

```diff
@@ -51,8 +51,14 @@ metricsService:
     targetPort: https
   type: ClusterIP
 auth:
-  secretName: aws-creds
-  createSecret: true
-  accessKeyId: ""
-  secretAccessKey: ""
-  sessionToken: ""
+  secrets:
+    # Specify the names of secrets in your cluster that contain a registries credentails.
+    # E.g.: https://github.com/spectrocloud-labs/validator/blob/main/chart/validator/templates/plugin-secret-oci.yaml
+    # - secretName: oci-registry-creds
+    # - secretName: ecr-registry-creds
+env:
+  # Specify any environment variables you'd like to have defined in your deployment
+  # - name: VARIABLE_1
```

Comment:
💬 I think we should include `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY`, all commented out. See https://pkg.go.dev/golang.org/x/net/http/httpproxy.

Reply:
We need to support custom CA certificates in addition to proxy env vars. That means custom volumes & volumeMounts.

Sample config:
```yaml
volumeMounts:
- name: ca-pemstore
  metadata:
    merge_strategy: replace
  mountPath: "/etc/ssl/certs/{{ .ProxyCaCertName }}"

volumes:
- name: ca-pemstore
  hostPath:
    path: "/usr/local/share/ca-certificates/{{ .ProxyCaCertName }}"
  metadata:
    merge_strategy: replace

OR

- name: ca-pemstore-2
  secret:
    secretName: ca-cert-secret
```

Additionally, we'd need an init container to execute `update-ca-certificates`, per https://stackoverflow.com/questions/52636213/docker-distroless-image-how-to-add-customize-certificate-to-trust-store.